shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -79,30 +56,6 @@ FlutterProjectArgs& EmbedderConfigBuilder::GetProjectArgs() {
   return project_args_;
 }
 
-void EmbedderConfigBuilder::SetSoftwareRendererConfig(SkISize surface_size) {
-  renderer_config_.type = FlutterRendererType::kSoftware;
-  renderer_config_.software = software_renderer_config_;
-  context_.SetupSurface(surface_size);
-}
-
-void EmbedderConfigBuilder::SetRendererConfig(EmbedderTestContextType type,
-                                              SkISize surface_size) {
-  switch (type) {
-    case EmbedderTestContextType::kOpenGLContext:
-      SetOpenGLRendererConfig(surface_size);
-      break;
-    case EmbedderTestContextType::kMetalContext:
-      SetMetalRendererConfig(surface_size);
-      break;
-    case EmbedderTestContextType::kVulkanContext:
-      SetVulkanRendererConfig(surface_size);
-      break;
-    case EmbedderTestContextType::kSoftwareContext:
-      SetSoftwareRendererConfig(surface_size);
-      break;
-  }
-}
```

Comment:
No need for for the `type` argument anymore since the `EmbedderConfigBuilder` already took an `EmbedderTestContext` of that type in its constructor, so we knew the type all along.